packages/console/src/pages/TenantSettings/Subscription/PlanQuotaTable/index.tsx

```diff
@@ -68,7 +68,7 @@ function PlanQuotaTable({ subscriptionPlans }: Props) {
         isRowHoverEffectDisabled
         rowGroups={quotaTableRowGroups}
         rowIndexKey="quotaKey"
-        rowClassName={(_, index) => conditional(index % 2 !== 0 && styles.colorRow)}
+        rowClassName={(_, index) => conditional(index % 2 === 0 && styles.colorRow)}
```

Comment:
Wait

Reply:
Per offline discussion with @fleuraly , now odd rows of the plan quota table should have a colored background.